src/com/ecfeed/core/generators/algorithms/ScoreBasedAlgorithmMock.java

```diff
@@ -0,0 +1,48 @@
+package com.ecfeed.core.generators.algorithms;
```

Comment:
Why do we even need this mock? To implement the 'constraintCheck' function? That should be done by simply providing our own constraint evaluator.